docs/source/FAQ/index.rst

```diff
@@ -67,73 +67,73 @@ Currently we don't have any special ranks in place.
 
 What are MyM's, what can I use them for and how do I get them?
 --------------------------------------------------------------
 
-MyM is the ingame currency used for the market, player to player trading, buying/selling Claimblocks... You can change tokens into MyM, but you can also make quite a lot from trading with other players.
+MyM is the ingame currency used for the market, player to player trading, buying/selling Claimblocks... You can change tokens into MyM, but you can also earn quite a lot from trading with other players.
 
 
 What are chunk loader tickets, what can I use them for and how do I get them?
 -----------------------------------------------------------------------------
 
-Chunk loader tickets are used to keep your chunk loaders active. All 1.7. servers introduce online (iron block) and offline (gold block) chunk loaders. The chunk loaders use up a certain amount of chunk loader tickets per minute, dependent on the size of the area you want to keep loaded. You get chunk loader tickets by changing tokens into them in the /shop.
+Chunk loader tickets are used to keep your chunk loaders active. All 1.7.x servers introduce online (iron block) and offline (gold block) chunk loaders. The chunk loaders use up a certain amount of chunk loader tickets per minute, dependent on the size of the area you want to keep loaded and on whether you are currently online or not. You get chunk loader tickets by exchanging tokens for them in the /shop.
 
 
 Tokens? MyMs? Tickets? I am confused, do you have more information?
 ------------------------------------------------------------------
 
-For a more detailed explanation of the different things you can get from voting, please have a look at `this thread <https://mineyourmind.net/forum/threads/figuring-out-the-way-new-voting-tickets-and-tokens-work.13054/>`_.
+For a more detailed explanation about benefits of voting, please have a look at `this thread <https://mineyourmind.net/forum/threads/figuring-out-the-way-new-voting-tickets-and-tokens-work.13054/>`_.
 
 
 What protection does this server use?
 -------------------------------------
 
-On the general server's we use GriefPrevention due to its ease of use for the player. Other servers with modpacks like agrarian skies or crash landing use worldguard regions created from the skyblock plugin. Both third party protection systems are extended by our custom ProtectEx extension to protect against more modded items. More information about Grief Prevention can be found :ref:`here <ref-griefprevention-tutorial>`
+Generally we use GriefPrevention due to its ease of use for the player. Other servers with modpacks like Agrarian Skies or Crash Landing use WorldGuard regions created from the skyblock plugin. Both third party protection systems are extended by our custom ProtectEx extension to protect against more modded items. More information about Grief Prevention can be found `here <http://docs.mym.li/en/latest/Knowledge_Base/griefprevention.html>`_.
```

Comment:
please keep the ref, else the link breaks once either the domain changes or the file.